addons/l10n_dk_oioubl/models/account_edi_xml_oioubl_201.py

```diff
@@ -0,0 +1,285 @@
+from odoo import models, tools
+
+DANISH_NATIONAL_IT_AND_TELECOM_AGENCY_ID = '320'
+
+PAYMENT_MEANS_CODE = {
+    # https://www.oioubl.info/codelists/en/urn_oioubl_codelist_paymentmeanscode-1.1.html
+    'unknown': 1,
+    'cash': 10,
+    'cheque': 20,
+    'debit': 31,
+    'bank': 42,
+    'card': 48,  # credit card
+    'direct debit': 49,
+    'compensation': 97,
+}
+UBL_TO_OIOUBL_TAX_CATEGORY_ID_MAPPING = {
+    # Simple mapping between tax type provided in UBL and what is accepted in OIOUBL
+    # https://docs.peppol.eu/poacc/billing/3.0/codelist/UNCL5305/
+    # https://www.oioubl.info/codelists/en/urn_oioubl_id_taxcategoryid-1.1.html
+    'AE': 'ReverseCharge',
+    'E': 'ZeroRated',
+    'S': 'StandardRated',
+    'Z': 'ZeroRated',
+    'G': 'ZeroRated',
+    'O': 'ZeroRated',
+    'K': 'ReverseCharge',
+    'L': 'ZeroRated',
+    'M': 'ZeroRated',
+}
+TAX_POSSIBLE_VALUES = set(UBL_TO_OIOUBL_TAX_CATEGORY_ID_MAPPING.values())
+ENDPOINTID_SCHEMEID = {
```

Comment:
Can't you use `EAS_MAPPING` in `account_edi_ubl_cii` ? There only seems to be a specific case for DK ("CVR").

Reply:
Agreed with Julien that the map was kind of useless in the end.
Let's make it with a simple if.